minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
```

Comment:
should be `./compose/certbot/production/Dockerfile` . don't forget to change the dockerfile path and add the one for the staging